### ISSUE TITLE
[Enhancement]Choose and customise your theme

### ISSUE THREAD

**Issue by warrendholmes** [Automattic/wc-calypso-bridge#1181]
Related p2 dicussion peapX7-2rb-p2#choose-and-customize-a-theme

There's a few things we want to do here:

_Edited by Ilyas_

### Acceptance Criteria

**WooCommerce:**

_Moved to its own issue_

**In the themes gallery:**

1. Only show Woo themes. There's some discussion here https://github.com/orgs/Automattic/projects/562/views/18?filterQuery=-status%3ADone+store&pane=issue&itemId=20916254, and I think we will need Serenity's help filtering out non-Woo themes.
1. The category filters should be removed.
1. Add a new upsell banner for users that want to upload their own theme
1. When a user activates a theme, they should see the congratulations page. For some reason, this isn't working on Woo Express sites but works fine on the Business plan or Ecommerce.
1. When the work is ready for review, please prepare a demo video to show peapX7-2rb-p2#comment-3438

**Comment by ilyasfoo** [comment]
> - Only show Woo themes. There's some discussion here https://github.com/orgs/Automattic/projects/562/views/18?filterQuery=-status%3ADone+store&pane=issue&itemId=20916254, and I think we will need Serenity's help filtering out non-Woo themes.
> - The category filters should be removed.
> - Add a new upsell banner for users that want to upload their own theme
> - When a user activates a theme, they should see the congratulations page. For some reason, this isn't working on Woo Express sites but works fine on the Business plan or Ecommerce.

@warrendholmes Apologies, I forgot to mention, I'm not sure yet if we can accomplish all of the above themes gallery changes without making changes to core. I was under the impression that we want to reposition, update copy, and update the redirect task only initially.

**Comment by warrendholmes** [comment]
@ilyasfoo I think the theme gallery is Calypso. Why would we need to make any changes in core?

**Comment by ilyasfoo** [comment]
@warrendholmes  Ah, I misunderstood. I thought that it was referring to WordPress core theme page 😓 . It should be possible then, I just need to dig into a few more details.

> Only show Woo themes. There's some discussion here https://github.com/orgs/Automattic/projects/562/views/18?filterQuery=-status%3ADone+store&pane=issue&itemId=20916254, and I think we will need Serenity's help filtering out non-Woo themes.

Based on what I'm seeing, these are all themes we have with a `WooCommerce` label: Tsubaki, Tazza, Zaino. I assume our goal is then to display only these themes (including anything upcoming with `WooCommerce` label)? Should users be able to override our filter to show all other themes?

> Add a new upsell banner for users that want to upload their own theme

I'm seeing this upsell on my free trial site. I assume we're going to replace it:

<img width="843" alt="image" src="https://github.com/Automattic/wc-calypso-bridge/assets/3747241/730fd992-b803-47d3-a06f-d41e951623e5">

>When a user activates a theme, they should see the congratulations page. For some reason, this isn't working on Woo Express sites but works fine on the Business plan or Ecommerce.

This sounds like a bug, perhaps we can address it in a follow-up issue instead.

**Comment by warrendholmes** [comment]
> @warrendholmes Ah, I misunderstood. I thought that it was referring to WordPress core theme page 😓 . It should be possible then, I just need to dig into a few more details.

No worries!

> Based on what I'm seeing, these are all themes we have with a WooCommerce label: Tsubaki, Tazza, Zaino. I assume our goal is then to display only these themes (including anything upcoming with WooCommerce label)? Should users be able to override our filter to show all other themes?

There should be two more : Thriving Artist https://wordpress.com/theme/thriving-artist and Amulet https://wordpress.com/theme/amulet

> I'm seeing this upsell on my free trial site. I assume we're going to replace it:

Yep, we want to change that to upgrade to the free trial https://hostedwoo.files.wordpress.com/2023/05/appearance-_-themes.png 

> Edit: Or, we can address it together since it's related. I've looked a bit into it, the congrats page is located at /marketplace/thank-you/{siteslug}?themes=tsubaki. I was able to manually navigate to it on my free trial site.

Yep, we definitely want to fix that in this issue.

**Comment by ilyasfoo** [comment]
I'll split this issue

**Comment by ilyasfoo** [comment]
Moved the WooCommerce overrides to https://github.com/Automattic/wc-calypso-bridge/issues/1195 so that it can be worked on in parallel

**Comment by ilyasfoo** [comment]
This was done a while ago
